code/modules/mob/living/carbon/carbon_procs.dm

```diff
@@ -350,7 +350,7 @@
 	var/obj/item/organ/internal/eyes/E = get_int_organ(/obj/item/organ/internal/eyes)
 	. = ..()
 
-	if((E && (E.status & ORGAN_DEAD)) || !. || E.is_broken())
+	if(E && ((E.status & ORGAN_DEAD) || E.is_broken()) || !.)
```

Comment:
```suggestion
	if((E?.status & ORGAN_DEAD) || E?.is_broken() || !.)
```
Should also work

Reply:
I think one sole nullcheck is probably fine